src/components/common/YourCompany/YourCompany.vue

```diff
@@ -193,23 +193,6 @@
       />
     </div>
 
-    <!-- Business Type (firm corrections only) -->
-    <template v-if="isFirmCorrectionFiling">
```

Comment:
So the ChangeBusinessType component (on the rhs) handles display and edit?

Reply:
Yes, that's right. It is reusable for rendering Business Type for correction filings, and we need it for SP/GP tooltips.